ramls/customField.json

```diff
@@ -28,6 +34,39 @@
       "type": "string",
       "description": "The description of the custom field",
       "example": "Provide a department"
+    },
+    "maxSize": {
+      "type": "integer",
+      "description": "The text field maximum length",
+      "example": 42
+    },
+    "default": {
+      "type": "boolean",
+      "description": "Defines if the custom field is default",
+      "example": false
+    },
+    "defaults": {
+      "type": "array",
+      "description": "Defines the custom field default value(s)",
+      "example": "ABC"
+    },
+    "multiSelect": {
```

Comment:
why is it that we only have a boolean for "mutiSelect" but not other types?

Reply:
<img width="1271" alt="Screen Shot 2019-08-21 at 12 28 10 PM" src="https://user-images.githubusercontent.com/37537790/63420351-2b35a400-c40f-11e9-9dcc-147f70daee01.png">
based on the definition `multiSelect` represents radio, single/muli select dropdowns

Reply:
ok - I got confused because the data model says "selectField" which could represent radio, single-select dropdown or multi-select dropdown options and in the json file, we named the field "multiSelect" - when I see multiSelect - the first thing that comes to my mind is multi-select dropdown. Do you think it will be a good idea to rename? Thanks!

Reply:
@sduvvuri-ebsco the schema for the custom fields will be updated as follows
```...
"textField": {
      "type": "object",
      "properties": {
        "maxSize": {
          "type": "integer",
          "description": "The text field maximum length",
          "example": 42
        }
      }
    },
 "checkBoxField": {
     "type": "object",
     "properties": {
        "default": {
          "type": "boolean",
          "description": "Defines if the custom field is default",
          "example": false
        }
      }
    },
 "selectField": {
     "type": "object",
     "properties": {
        "defaults": {
          "type": "array",
          "description": "Defines the custom field default value(s)",
          "example": "ABC"
        },
        "multiSelect": {
          "type": "boolean",
          "description": "Defines if the custom field is a multi select",
          "example": false
        },
        "options": {
          "type": "object",
          "description": "Defines the options for the custom field",
          "$ref": "customFieldOptions.json"
        }
      }
    }
...```